Library/AppEnvironmentTests.swift

```diff
@@ -60,19 +60,23 @@ final class AppEnvironmentTests: XCTestCase {
     XCTAssertNil(AppEnvironment.current.currentUser)
```

Comment:
Super minor thing. Do we want to check the `analyticsIdentityData` here to be `nil`?

Reply:
oh sure, let me add that 👍